filament/backend/src/metal/MetalHandles.mm

```diff
@@ -888,6 +889,9 @@ BufferDescriptor b(nullptr, 0u, [](void* buffer, size_t size, void* user) {
                 descriptor.stencilAttachment.resolveLevel = depthStencilAttachment.level;
                 descriptor.stencilAttachment.resolveSlice = depthStencilAttachment.layer;
                 descriptor.stencilAttachment.storeAction = MTLStoreActionMultisampleResolve;
+                descriptor.stencilAttachment.stencilResolveFilter = (context->supportsStencilResolve)
+                    ? MTLMultisampleStencilResolveFilterDepthResolvedSample
+                    : MTLMultisampleStencilResolveFilterSample0;
```

Comment:
Given that both cases effectively filter the zeroth sample, should we bother with this at all now? (I mean, with everything related to the stencil resolve support query). At this point we don't actually use those `.apple5`+ family features. Might come the time when we'll actually change the resolve filter, but until then I feel it's more of a maintenance burden.

Reply:
All the currently unused parts are removed in https://github.com/shapr3d/filament/pull/67/commits/fe2c820a5e337a01ec4874c5be0eee4e0d61e25c